PowerEditor/src/ScitillaComponent/Buffer.h

```diff
@@ -122,7 +122,18 @@ class FileManager final
 		int _encoding;
 		EolType _eolFormat;
 	};
+
+	FileManager() = default;
 	~FileManager();

```

Comment:
`~FileManager() = default;` ?

Reply:
FileManager dtor can't be made default as user defined dtor already present which de-allocate a vector of `Buffer`.